enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -203,80 +205,49 @@ metadata:
 
 How-to references:
- * [import cert via powershell](https://docs.microsoft.com/en-us/powershell/module/pki/import-certificate?view=windowsserver2019-ps)
- * [delete cert via powershell](https://stackoverflow.com/questions/37228851/delete-certificate-from-computer-store)
-
---
+* [import cert via powershell](https://docs.microsoft.com/en-us/powershell/module/pki/import-certificate?view=windowsserver2019-ps)
+* [delete cert via powershell](https://stackoverflow.com/questions/37228851/delete-certificate-from-computer-store)
 
-Also note that there is another way to get the data required in step 2 rather than accessing the ConfigMap directly,
-  but I prefer not to follow it. For completeness, I will list it out here.
-  * Update the operator’s Deployment to support trusted CA injection by mounting the trusted CA ConfigMap
-  ```yaml
-  apiVersion: apps/v1
-  kind: Deployment
-  metadata:
-    name: windows-machine-config-operator
-    namespace: openshift-windows-machine-config-operator
-    annotations:
-      config.openshift.io/inject-proxy: windows-machine-config-operator
-  spec:
-      ...
-        containers:
-          - name: windows-machine-config-operator
-            volumeMounts:
-            - name: trusted-ca
-              mountPath: /etc/pki/ca-trust/extracted/pem
-              readOnly: true
-        - name: trusted-ca
-          configMap:
-            name: trusted-ca
-            items:
-              - key: ca-bundle.crt
-                path: tls-ca-bundle.pem
-  ...
-  ```
-  * Create a file watcher that watches changes to the mounted trust bundle that kills the main operator process and
-    allows the backing k8s Deployment to start a new Pod that mounts the updated trust bundle. 
-    Implementation example: [cluster-ingress-operator](https://github.com/openshift/cluster-ingress-operator/pull/334)
-  * But then the same concern about avoiding unnecessary reconciliation remains, as in the [previous section](#configuring-custom-trusted-certificates): 
-    how to detect if the trust bundle file changed or the operator just restarted for another reason?
 ---
 
 ### Test Plan & Infrastructure Needed
 
-In addition to unit testing individual WMCO packages and controllers, an e2e will be added to the WMCO repo for the
-master/release-4.14 branches. We will leverage the [existing proxy workflow](https://steps.ci.openshift.org/workflow/openshift-e2e-aws-proxy)
-from the release repo and add a job to the WMCO repo's CI config to test with. We will run regression testing to make
-sure the addition of the egress proxy feature does not break existing functionality. If we release a community offering
-with this feature, we will need to add a CI job using a cluster-wide proxy on OKD.
+In addition to unit testing individual WMCO packages and controllers, an e2e job will be added to the release repo for
+WMCO's master/release-4.14 branches. This can replace the current vSphere job to avoid growing the number of required jobs.
```

Comment:
vSphere is the number 1 platform for Windows nodes. I suggest not replacing this. Or do you think the vSphere platform:none job covers that?

Reply:
I will let @openshift/openshift-team-windows-containers weigh in here